content/docs/dvclive/api-reference/live/next_step.md

```diff
@@ -29,6 +29,18 @@ You can use `Live.next_step()` to increase the `step` by 1 (one).
 In addition to increasing the `step` number, it will call `Live.make_report()`
 and `Live.make_summary()`.
 
+If `save_dvc_exp=True` has been passed to `Live()`, `Live.next_step()` will
```

Comment:
See suggestions in `live.end()`. It might also be worth mentioning that this is useful for live monitoring.